packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -7,21 +7,21 @@ const DEFAULT_THUMB_SIZE = 12;
 
 export const getStyles = (props: IToggleStyleProps): IToggleStyles => {
   const { theme, className, disabled, checked, inlineLabel, onOffMissing } = props;
-  const { semanticColors, palette } = theme;
+  const { semanticColors } = theme;
 
   // Tokens
   const pillUncheckedBackground = semanticColors.bodyBackground;
   const pillCheckedBackground = semanticColors.inputBackgroundChecked;
   // TODO: after updating the semanticColors slots mapping this needs to be semanticColors.inputBackgroundCheckedHovered
-  const pillCheckedHoveredBackground = palette.themeDark;
-  const thumbUncheckedHoveredBackground = palette.neutralDark;
+  const pillCheckedHoveredBackground = semanticColors.primaryButtonBackgroundPressed;
```

Comment:
This is EXACTLY the problem with semantic colors. We are using primarybutton tokens for background, and actionbutton tokens for foreground colors, defeating the purpose of semantic colors being a way to enforce contrast. ActionButtons expect to be on top of actionLinkBackground. So there is no way we could validate contrast with actionlink foregrounds on button backgrounds.

It also is unclear for a designer what to do. Asking a dev to ask a designer, who doesn't understand what's the "right thing" seems like bad advice, unless designers have been trained in a process that I'm unaware of.

What would you describe is the right thing for the designer to say?

Perhaps in the semantic colors model, one might say: "A toggle needs its own set of colors". So we'd add `toggleBackground`, `toggleText`, etc. Obviously this will not scale.

To scale, perhaps we use `input` colors when unchecked, and `primarybutton` colors when checked. As long as we use the same category of bg/fg colors, we could ensure accessibility.

But I do think the better method here is to define a recipe for the color shades, use a smaller list of variables, and opt into a recipe.